Userland/Games/Snake/Game.cpp

```diff
@@ -57,25 +68,28 @@ ErrorOr<NonnullRefPtr<Game>> Game::try_create()
     TRY(food_bitmaps.try_ensure_capacity(food_bitmaps_files.size()));
 
     for (auto file : food_bitmaps_files) {
-        auto bitmap = Gfx::Bitmap::try_load_from_file(file);
-        if (bitmap.is_error()) {
-            dbgln("\033[31;1mCould not load bitmap file\033[0m '{}': {}", file, bitmap.error());
-            return bitmap.release_error();
-        }
-
-        food_bitmaps.unchecked_append(bitmap.release_value());
+        auto bitmap = TRY(load_bitmap(file));
+        food_bitmaps.unchecked_append(bitmap);
     }
 
+    return food_bitmaps;
+}
+
+ErrorOr<NonnullRefPtr<Game>> Game::try_create()
+{
+    auto food_bitmaps = TRY(load_food_bitmaps());
+
     return adopt_nonnull_ref_or_enomem(new (nothrow) Game(move(food_bitmaps)));
 }
 
 Game::Game(NonnullRefPtrVector<Gfx::Bitmap> food_bitmaps)
     : m_food_bitmaps(move(food_bitmaps))
 {
+    m_skin = Config::read_string("Snake"sv, "Snake"sv, "Skin"sv, m_skin);
+    MUST(load_snake_bitmaps());
```

Comment:
This loading wants to also happen in `Game::try_create()` so that we can `TRY()` it.